kubernetes/helm_charts/core/nginx-public-ingress/values.j2

```diff
@@ -59,6 +59,7 @@ resources:
 
 repository: {{proxy_repository|default('proxy')}}
 image_tag: {{ image_tag }}
+annotations: {{ public_nginx_annotations | to_json }}
```

Comment:
what if the public_nginx_annotaitions are empty?
`public_nginx_annotaions | d('{}') | to_json`